### ISSUE TITLE
Shortcuts for text formatting

### ISSUE THREAD

**Issue by rsarky** [vimwiki/vimwiki#948]
It would be useful to have shortcuts (probably in normal mode) that format the highlighted text into bold, italics etc.
I am not sure if this already exists, couldnt find it from a quick search.

**Comment by brennen** [comment]
[vim-surround](https://github.com/tpope/vim-surround) might be of interest.  With a visual selection, you could hit `S*` to bold, for example.

**Comment by rsarky** [comment]
Yes, I have heard of that. But I was a bit reluctant to install a plugin for this usecase. Moreover vimwiki already has support for other shortcuts for eg. the header shortcuts, can this fit into the existing infrastructure?

**Comment by tinmarino** [comment]
> can this fit into the existing infrastructure?

Yes ! This is actually a good idea. Plus devs "only" need to copy paste the surround plugin code (because vimwiki has only vim as requirements).
A pull request would be welcome.

**Comment by rsarky** [comment]
Hey @tinmarino. I can pick this unless you already have started on this?

**Comment by tinmarino** [comment]
Hi @rsarky, I did not code anything already and ... it was not in mu priority list. I Assigned to me as a reminder because I personaly find the feature interesting.
Don't hesitate to ask here for some advices.
I would map by default:
```
map_prefix = <leader>ws or <leader>s or gs 
<map_prefix>b = bold()
<map_prefix>i = italic()
<map_prefix>c = code()
```

Thanks for the collaboration

**Comment by uolot** [comment]
Could this be disabled by default?

**Comment by folofjc** [comment]
Hi @tinmarino. Is there an update on this? I came here looking just for this!

**Comment by systemb4** [comment]
Any update on this? Is this something we have to submit as a pull request or can the developers implement this? (very useful idea by the way)
